src/languages/_english.json

```diff
@@ -1428,9 +1430,13 @@
       }
     },
     "transaction_details": {
+      "from": "From",
+      "to": "To",
       "value": "Value",
       "hash": "Tx Hash",
-      "network_fee": "Network Fee"
+      "network_fee": "Network Fee",
+      "hash_copied": "\uDBC0\uDC63 Transaction Hash Copied",
```

Comment:
nit - just wanted to confirm `hash_copied` and `address_copied` are supposed to have that capitalization on each word (vs just the first word, which seems to be the pattern for our toasts content). Didn't see this defined on the figma anywhere.